src/cpp/src/tokenizer.cpp

```diff
@@ -455,7 +455,7 @@ class Tokenizer::TokenizerImpl {
 
     TokenizedInputs encode(std::vector<std::string>& prompts, const ov::AnyMap& tokenization_params = {}) {
         OPENVINO_ASSERT(m_ireq_queue_tokenizer, "Either openvino_tokenizer.xml was not provided or it was not loaded correctly. "
-                                                "Tokenizer::encode is not available");
+                        "Tokenizer::encode is not available");
```

Comment:
There are other asserts written with such indentation in this file. Revert it.

Reply:
Reverted